### PR TITLE
Java sources in Kotlin-As-Java plugin should have original property v…

### PR DESCRIPTION
**Pull request by MarcinAman**
…isibilites

### CODE REVIEW

**Comment by sellmair** [comment]
This needs adjustments after rebasing 🙄

File: plugins/base/build.gradle.kts [hunk 1]
```diff
@@ -4,6 +4,8 @@ plugins {
     id("com.jfrog.bintray")
 }
 
+val testUtils by configurations.creating
```
Comment (sellmair) [inline plugins/base/build.gradle.kts:7]: See #1055 for discussion regarding this idea!

Comment (MarcinAman) [reply plugins/base/build.gradle.kts:7]: Yes, i took it from that PR with hope that it is a good idea xd Once this PR will be resolved i'll rebase my changes

Comment (MarcinAman) [reply plugins/base/build.gradle.kts]: Ok, i've rebased this PR on recent dev changes to get rid to this configuration

**Pull request merged**